lilypad/lilypad-aurora-testnet/overview.md

```diff
@@ -1,27 +1,63 @@
----
-cover: ../.gitbook/assets/aurora record player.png
-coverY: -9.129886506935662
-layout:
-  cover:
-    visible: true
-    size: full
-  title:
-    visible: true
-  description:
-    visible: true
-  tableOfContents:
-    visible: true
-  outline:
-    visible: true
-  pagination:
-    visible: true
----
-
-# Overview
-
-
-
-{% embed url="https://youtu.be/yQnB2Yxia4Y?si=x3EOrTy4M_cSao7c" %}
-Lilypad: Internet-scale Distributed Compute - web3's new frontier
-{% endembed %}
+  ---
+  cover: ../.gitbook/assets/aurora record player.png
+  coverY: -9.129886506935662
+  layout:
+    cover:
+      visible: true
+      size: full
+    title:
+      visible: true
+    description:
+      visible: true
+    tableOfContents:
+      visible: true
+    outline:
+      visible: true
+    pagination:
+      visible: true
+  description: Trustless Distributed Compute for web3
+  ---
 
+  # Overview
+
+  ### What is Lilypad?
+  Lilypad is a verifiable trustless decentralized compute network. It aims to enable mainstream web3 applications by providing global, permissionless access to compute power. Lilypad leverages decentralized physical infrastructure networks like Filecoin to create an open compute network that is more accessible, efficient, and transparent. It addresses the challenges surrounding access to AI models and hardware by offering decentralized AI capabilities. Lilypad has recently released Lilypad V2 (Aurora) and has plans to go multichain and launch an incentivized test net.
```

Comment:
Hmm it's not quite right to say we address the challenges of access to AI models. I don't think we currently do.
We are trying to make access to the powerful compute hardware (needed for AI processing) easier though.